### ISSUE TITLE
Disabled and multiple=true does not work properly

### ISSUE THREAD

**Issue by corrspt** [cibernox/ember-power-select#96]
When we set  `disabled=true` and `multiple=true` the component still renders the input box and allows typing on it. Although the search box does not appear :+1: 
![ember-power-select](https://cloud.githubusercontent.com/assets/789263/10907732/9feaa34e-8224-11e5-900e-a8568ba64bc0.png)

```handlebars
{{#power-select options=names allowClear=true disabled=fieldsDisabled  selected=name onchange=(action (mut name)) multiple=true as |name|}}
    {{name}}
{{/power-select}}
```

This is the html generated when multiple and disabled are true

![select-html](https://cloud.githubusercontent.com/assets/789263/10907783/04c4fc42-8225-11e5-89c8-8bfe3a96ccbe.png)

**Comment by cibernox** [comment]
The input should be disabled too, this is clearly a bug.

**Comment by corrspt** [comment]
Thanks a lot @cibernox, great job! :)

**Comment by cibernox** [comment]
I've just published 0.3.8

**Comment by corrspt** [comment]
Just updated, thanks a million :+1:
